From: KevinDuan <duankaiwen@kylinos.cn>
Date: Mon, 17 Aug 2026 06:42:13 +0000
Subject: Translated using Weblate (Japanese)

Currently translated at 100.0% (2 of 2 strings)

Translation: openkylin-huanghe/peony-send-to-device
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-send-to-device/ja/
---
 peony-send-to-device/translations/peony-send-to-device_ja.ts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/peony-send-to-device/translations/peony-send-to-device_ja.ts b/peony-send-to-device/translations/peony-send-to-device_ja.ts
index ca00784..7dc618c 100644
--- a/peony-send-to-device/translations/peony-send-to-device_ja.ts
+++ b/peony-send-to-device/translations/peony-send-to-device_ja.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../send-to-device-plugin.cpp" line="321"/>
         <source>Send to a removable device</source>
-        <translation type="unfinished"></translation>
+        <translation>モバイルデバイスに送信</translation>
     </message>
 </context>
 <context>
@@ -14,7 +14,7 @@
     <message>
         <location filename="../send-to-device-plugin.h" line="48"/>
         <source>Send to a removable device</source>
-        <translation type="unfinished"></translation>
+        <translation>モバイルデバイスに送信</translation>
     </message>
 </context>
 </TS>
